docs/features/global_tools.md

```diff
@@ -34,6 +31,35 @@ exposed = { python310 = "python" } # (3)!
 2. The exposed binaries are the ones that will be available in the system path. `vim` has multiple and all of them are exposed.
 3. Here python is exposed as `python310` to avoid conflicts with other python installations. You can give it any name you want.
 
+### Manifest locations
+
+The manifest can be found at the following locations depending on your operation system.
+
+=== "Linux"
```

Comment:
wdyt about using heading or bold "Linux" here?

like

### Linux

or 

**Linux**

I think it looks more easy to read

Reply:
This should be aligned with the other tabbed headings so it stays selected, If you select `linux` in one page it will be remembered over all pages.

Reply:
@nichmor Did you run `pixi run docs`? This syntax is mkdocs specific